addons/gdUnit3/src/update/GdMarkDownReader.gd

```diff
@@ -0,0 +1,232 @@
+class_name GdMarkDownReader
```

Comment:
Isn't this feature creep? Why do we need a MarkDown reader in a test framework?

- You can open a browser to read the documents
- I can read markdown myself
- it is a plugin in itself
- you can advise to install it then use it when available

Reply:
I need it to show the latest release info in the update popup.
I wan't to have the full view of release information grabed from github.

The updater will be fully integrated in the plugin to be notified on each startup for possible update.
This will help users to get a automated clean update of the GdUnit API.